File: Java/Biblioteca.java
public class Biblioteca {
    private String titulo;
    private String autor;
    private int anoPublicacao;
    private int copiasDisponiveis;

    public Biblioteca(String titulo, String autor, int anoPublicacao, int copiasDisponiveis) {
        this.titulo = titulo;
        this.autor = autor;
        this.anoPublicacao = anoPublicacao;
        this.copiasDisponiveis = copiasDisponiveis;
    }

    public String obterTitulo() {}

    public String obterAutor() {}

    public int obterAnoPublicacao() {}

    public int obterCopiasDisponiveis() {}

    public boolean verificarDisponibilidadeCopia() {}

    public void obterInformacoes() {}

    public void emprestarCopia() {}

    public void devolverCopia() {}
}

File: C/Biblioteca.c
#include <stdbool.h>

typedef struct {
    char titulo[100];
    char autor[100];
    int anoPublicacao;
    int copiasDisponiveis;
} Biblioteca;

Biblioteca* criarBiblioteca(char titulo[], char autor[], int anoPublicacao, int copiasDisponiveis);
char* obterTitulo(Biblioteca* livro);
char* obterAutor(Biblioteca* livro);
int obterAnoPublicacao(Biblioteca* livro);
int obterCopiasDisponiveis(Biblioteca* livro);
bool verificarDisponibilidadeCopia(Biblioteca* livro);
void obterInformacoes(Biblioteca* livro);
void emprestarCopia(Biblioteca* livro);
void devolverCopia(Biblioteca* livro);
